conf/backup_method

```diff
@@ -46,7 +46,20 @@ This is an automated message from your beloved YunoHost server." | /usr/bin/mail
     # About the {now} placeholder:
     # https://borgbackup.readthedocs.io/en/stable/usage/create.html#description
     # In the archive name, you may use the following placeholders: {now}, {utcnow}, {fqdn}, {hostname}, {user} and some others.
-    "$borg" create --stats "::${name}-{now}" "$work_dir"
+    "$borg" create --stats "::${name}-{now}" "$work_dir" || status=$?
+    # Check whether the above command exit with a return code > 0
+    case "$status" in
+      "") # return code was 0, skip
+        ;;
+      "1")
+        # This is a warning, the script may continue, but let's warn the admin.
+        # See: https://github.com/borgbackup/borg/issues/6989#issuecomment-1223921392
+        echo "Some warning were logged during the backup, you'd probably check whether it was severe or not" >&2
+        ;;
+      *)
+        exit $status
+        ;;
+    esac
```

Comment:
This seems kinda convoluted ... What about just for checking the return code of the command ... what about : 

```suggestion
    "$borg" create --stats "::${name}-{now}" "$work_dir" || retcode=$?
    # Check whether the above command exit with a return code > 0
    if [[ "${retcode:-}" == 1 ]]
    then
        # This is a warning, the script may continue, but let's warn the admin.
        # See: https://github.com/borgbackup/borg/issues/6989#issuecomment-1223921392
        echo "Some warning were logged during the backup, you'd probably check whether it was severe or not" >&2
    elif [[ "${retcode:-}" != "" ]]
        exit $retcode
    fi
```